src/jbi/router.py

```diff
@@ -36,7 +36,7 @@ def get_whiteboard_tag(
     whiteboard_tag: Optional[str] = None,
 ):
     """API for viewing whiteboard_tags"""
-    data = configuration.get_yaml_configurations()
+    data = configuration.get_yaml_configurations().dict()["actions"]
```

Comment:
Maybe this helper on the configuration module should return the dict directly

Reply:
I did that initially but I was wondering whether we want to be using objects or dictionary's to access the underlying data. 

I can migrate this to a helper for the time being.